src/components/data/_tests/data-utils.test.js

```diff
@@ -428,3 +429,304 @@ describe('prepareAttributeForUpload', () => {
     })
   })
 })
+
+describe('AttributeTypeInput', () => {
+  it('renders radio buttons for available types', () => {
```

Comment:
This looks suspiciously like it is testing React's ability to render, which may be a valid thing to test, but I wouldn't do so for each component. I would add a comment here that answers the question, "What value am I getting for the cost of having to update the code in two places if I want to add another option?" Example: "Some options are conditional, so this ensures only the default options are present."

Reply:
🤔 Things like this might be better served by Jest's [snapshot testing](https://jestjs.io/docs/snapshot-testing), which would reduce the cost of having to update the test code. I think tests like these function better as documentation than snapshot tests. But, as with comments, I suppose there's an argument to be made here that the intent is obvious enough that it doesn't need explanation.